src/lib/plugins/behavior.js

```diff
@@ -1,9 +1,9 @@
-const Behavior = require("flying-squid").Behavior;
```

Comment:
yes you should have left it as is, it works thanks to require-self, and it is useful to make internal plugins normal plugins (ie you can just copy paste the file somewhere else)